tests/streamlit_ui_tests/streamlit.spec.ts

```diff
@@ -457,3 +457,51 @@ test.describe('Keyboard Shortcuts', () => {
     await expect(mito.locator('.endo-column-header-container-selected')).toHaveCount(1);
   });
 });
+
+test.describe('Context Menus', () => {
+  test('Column Header', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await expect(mito.locator('.mito-dropdown')).not.toBeVisible();
+    await mito.getByTitle('Column1').click({ button: 'right' });
+    await expect(mito.locator('.mito-dropdown')).toBeVisible();
```

Comment:
Can you actually check the right header is open -- e.g. just checking that the first action is available? I think it makes sense to do this for each of these.